packages/mask/src/plugins/Avatar/SNSAdaptor/index.tsx

```diff
@@ -25,6 +25,15 @@ const sns: Plugin.SNSAdaptor.Definition = {
                 RenderEntryComponent(EntryComponentProps) {
                     const [open, setOpen] = useState(false)
                     const clickHandler = () => setOpen(true)
+                    useEffect(
+                        () => () => {
```

Comment:
What's this for? Register a listener on tear-down an effect?

Reply:
@Lanttcat  `CrossIsolationMessages.events.requestOpenApplication.on`  will return an unsubscribe function.

Just return the expression

```ts
useEffect(() => {
    return CrossIsolationMessages.events.requestOpenApplication.on(fn)
}, [])
```